pkg/config/ingress.go

```diff
@@ -27,9 +32,10 @@ type IngressConfig struct {
 	// List of IP addresses and NIC names where the router will be listening. The NIC
 	// names get translated to all their configured IPs dynamically. Defaults to the
 	// configured IPs in the host at MicroShift start.
-	ListenAddress      []string `json:"listenAddress"`
-	ServingCertificate []byte   `json:"-"`
-	ServingKey         []byte   `json:"-"`
+	ListenAddress            []string `json:"listenAddress"`
+	ServingCertificate       []byte   `json:"-"`
+	ServingKey               []byte   `json:"-"`
+	ServingCertificateSecret string   `json:"certificateSecret"`
```

Comment:
Missing description? Not sure what allowed/expected values are

Reply:
added  description